datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -55,11 +55,12 @@ python3-gearman==0.1.0; sys_platform != 'win32' and python_version > '3.0'
 pytz==2019.3
 pyvmomi==v6.5.0.2017.5-1
 pywin32==227; sys_platform == 'win32'
-pyyaml==5.1
+pyyaml==5.3
```

Comment:
No need, https://github.com/DataDog/integrations-core/pull/6043

Reply:
reverted and cherry-picked from #6043